h3i/src/actions/h3.rs

```diff
@@ -105,11 +106,12 @@ pub enum Action {
 }
 
 /// Configure the wait behavior for a connection.
+#[serde_as]
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum WaitType {
     /// Wait for a time before firing the next action
-    WaitDuration(Duration),
+    WaitDuration(#[serde_as(as = "serde_with::DurationMilliSeconds")] Duration),
```

Comment:
In order to maintain consistency with other qlog timestamp fields, should this be https://docs.rs/serde_with/latest/serde_with/struct.DurationMilliSecondsWithFrac.html ?

Reply:
idk, you tell me :laughing: I changed it to `DurationMilliSecondsWithFrac<f64>` which incidentally also makes the post-fmt code more readable.